File: Website/src/Models/Region.ts
export class Region {
  index:number;
  regionType:string;
  description: string;
  topLeft: Point;
  width:number;
  height:number;
  // bottomRight: Point;
}

export class Point {
  x: number;
  y: number;
}


export class ModalInfo {
  header: string;
  textBinding: string;
  index: number;
}

export class ModalEmailInfo {
  header: string;
  to: string;
  subject: string;
  message: string;
}

File: src/OCR_API/Models/Region.cs
namespace OCR_API.Models
{
    public class Region
    {
        public int Index { get; set; }
        public string RegionType { get; set; }
        public string Description { get; set; }
        public Point TopLeft { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
    }
    public class Point
    {
        public decimal X { get; set; }
        public decimal Y { get; set; }
    }
}
